src/tiledb/cloud/client.py

```diff
@@ -115,10 +136,12 @@ def login(
         "api_key": {},
     }
 
-    # Is user logs in with username/password we need to create a session
+    # If user logs in with username/password we need to create a session
     if (token is None or token == "") and not no_session:
         config.setup_configuration(**config_args)
         client.set_threads(threads)
+
+        # TODO: future migration to new server API.
```

Comment:
I usually recommend adding a Shortcut story to TODOs because otherwise TODOs tend to not be actionable due to the lack of context. The server's code base has plenty of TODOs that nobody can act on anymore 😕